core/src/main/scala/akka/kafka/CommitTimeoutException.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2014 - 2016 Softwaremill <http://softwaremill.com>
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.kafka
+
+/**
+ * Calls to `commitJavadsl` and `commitScaladsl` will be failed with this expcetion if
+ * Kafka doesn't respond within `commit-timeout`
+ */
+class CommitTimeoutException(msg: String) extends Exception(msg) {}
```

Comment:
If we have it extend `TimeoutException` it would be source compatible.

Reply:
:+1: